src/Beberlei/Metrics/Collector/Collector.php

```diff
@@ -18,9 +18,38 @@
  */
 interface Collector
 {
+    /**
+     * Update one or more counters by some arbitrary amount
+     *
+     * @param string $variable
+     * @param int $value The amount to increment the counter by
+     */
+    public function measure($variable, $value);
+
+    /**
+     * Increment a counter
```

Comment:
That's nitpicking... but because I try to be very meticulous, I'll fix it.
Thanks!